webhook.go

```diff
@@ -174,8 +175,73 @@ func loadtsiMutateConfig(configFile string) (*tsiMutateConfig, error) {
 	return &cfg, nil
 }
 
+// isSafe validates the format of the request to check if the requested
+// operation is permitted. For CREATE operations, it checks if the request
+// contains references to proteced socket and /etc Volumes.
+// And since UPDATE, does not allowed volume modifications, it only makes sure
+// that sidecar image has not been modified
+// it returns TRUE if operation is safe or FALSE with error explanation otherwise
+func isSafe(pod *corev1.Pod, operationType string) (bool, error) {
+
+	// this output can be used for creating tests/FakeIsSafeXXX.json files
+	glog.Infof("isSafe log. Operation %v", operationType)
+	logJSON("FakeIsSafeX.json", pod)
+
+	result := errors.New("")
```

Comment:
I read later on how you are using error. more like using a message.... I think if using error, the code should not confound it with regular uses of error. I think it can get very confusing. 

Perhaps the return here should be `bool, string, err` or just `bool, string` if it is not possible for the function to error.